spec.html

```diff
@@ -273,6 +273,40 @@ <h1>Runtime Semantics: BindingClassDeclarationEvaluation</h1>
         <p><emu-grammar>ClassDeclaration : `class` ClassTail</emu-grammar> only occurs as part of an |ExportDeclaration| and the setting of a name property and establishing its binding are handled as part of the evaluation action for that production. See <emu-xref href="#sec-exports-runtime-semantics-evaluation"></emu-xref>.</p>
       </emu-note>
     </emu-clause>
+
+  <emu-clause id="sec-coalesce-getter-setter" aoid=CoalesceGetterSetter>
+    <h1>CoalesceGetterSetter ( _element_, _other_ )</h1>
+    <emu-alg>
+      1. Assert: IsAccessorDescriptor(_other_.[[Descriptor]]) and IsAccessorDescriptor(_element_.[[Descriptor]]) are both *true*,
+      1. If _element_.[[Descriptor]] has a [[Get]] field,
+        1. Set _other_.[[Descriptor]].[[Get]] to _element_.[[Descriptor]].[[Get]].
```

Comment:
Is it worth asserting that other does not have a Get field?

Reply:
That assertion might not be valid for public methods (where we don't have the benefit of early errors for misuse in conjunction with computed property names).